### PR TITLE
avir: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **avir/3.0**

avir is header only library for image resizing.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]

**Comment by toge** [comment]
I fixed same miss of https://github.com/conan-io/conan-center-index/pull/8007.

**Review by uilianries — APPROVED** [review]

File: recipes/avir/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import os
+
+from conans.errors import ConanInvalidConfiguration
+from conans import ConanFile, CMake, tools
```
Comment (prince-chrismc) [inline recipes/avir/all/conanfile.py:4]:
```suggestion
from conans import ConanFile, tools
```

**Review by prince-chrismc — APPROVED** [review]
Not worth blocking but small detail

File: recipes/avir/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/avir/all/conanfile.py:4]: Thank you!
I will fix it at avir future release.

File: recipes/avir/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy("*.h", dst="include", src=self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.filenames["cmake_find_package"] = "avir"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "avir"
```
Comment (SpaceIm) [inline recipes/avir/all/conanfile.py:33]: same than https://github.com/conan-io/conan-center-index/pull/8029#discussion_r750657031, upstream doesn't install CMake config files, so no need to enforce something.

**Pull request merged**